README.md

```diff
@@ -3,19 +3,61 @@ Clapper
 
 Synchronize setup and deployment for the Director.
 
-Configuration Validation
-========================
+Ansible-based Validations
+-------------------------
+
+Recently, we focused on writing automated validations that require
+little human setup or interference. These are driven by
+[Ansible](http://ansible.com/) we plan to provide an API that will let
+us use these from the TripleO web UI and command line clients.
+
+### Running a validation
+
+For now, you need to run them manually. All the validations live in
+the [ansible-tests/validations](ansible-tests/validations) directory.
+
+To run a validation you need to:
+
+    $ git clone https://github.com/rthallisey/clapper.git
+    $ cd clapper/ansible-tests
+    $ ls validations  # pick a validation to run
+    $ ansible-playbook -v -i tripleo-ansible-inventory.py validations/some_validation.yaml
+
+NOTE: some validations will provide useful information that can cause
+trouble but isn't in an of itself enough to fail the check. Most
```

Comment:
I see. In that case, you can simply say that some of the validations only produce useful output when run with a higher verbosity, and their status are not relevant, the deployer really needs to check their output.